### ISSUE TITLE
Robo Recall, Crash on mission 2.1 

### ISSUE THREAD

**Issue by SirRagesAlot** [LibreVR/Revive#536]
Hey guys, I'm having some trouble running Robo Recall on Mission 2-1, the game just crashes everytime I load. 

I've tried reinstalling revive and the game on oculus but to no avail. I was wondering if anyone else had similar issues or has a solution. 

Thanks

**Comment by Armada651** [comment]
Can you try https://github.com/LibreVR/RoboRevive, so we know whether it is a problem with Revive or the game?

**Comment by bernardb13** [comment]
Hi, I have crashing issues with Robo Recall as well, but it crashes seemingly randomly not in any particular mission.  It happens with Revive and also with RoboRevive.  I get a crash reporter window on my desktop screen indicating that Unreal Engine has crashed.  I run SuperHot, Bullet Train, etc all with no problems at all.  Any ideas what could be causing this?

My setup:  Asus Z170, i7-6700K, 32GB RAM, EVGA 1080 HydroCopper, 256GBG M.2 System Drive, Samsung EVO 1 TB SSD Content drive, HTC Vive, SteamVR, Windows 10

Thanks for the help!

**Comment by Armada651** [comment]
Can you also try to see if you can reproduce the crash with RoboRevive?

**Comment by bernardb13** [comment]
Yes it crashes with RoboRevive as well.  I am at work now and don't have access to the crash log at the moment but I can send those if that might help.

**Comment by Armada651** [comment]
Oh btw, you should not start RoboRecall from the Revive Dashboard after installing the mod. If you do it will use Revive anyway instead of the mod.

**Comment by bernardb13** [comment]
Ahhh....will try that when I get home!

**Comment by bernardb13** [comment]
OK... I ran mission 1-2 completely with no crash - a very encouraging sign.  Did not have the time yet to try other missions - keeping fingers crossed.

Do you know if this is compatible with the Locomotion V6 mod (MGSStudios)?  Full locomotion plus no crashes would make Robo Recall my absolute favorite VR experience.

Anyway, will run other mission later today and update you.

**Comment by bernardb13** [comment]
Ran another couple of missions with no glitches whatsoever.  Thank you!  I guess there is something about my computer that is incompatible with the regular Revive.  RoboRevive works perfectly.

Am going to try running Robo Recall with the locomotion mod now and see if that breaks it.

**Comment by Armada651** [comment]
@bernardb13 Why do you think it's just your computer? The reason I asked you to test RoboRevive is to confirm it's not a problem with the game itself...

So logically, the crash should be in Revive.

**Comment by bernardb13** [comment]
I say that because I assume there are many other users out there successfully running Robo Recall using Revive with no crashes...
